docs/USAGE-GUIDE.md

```diff
@@ -399,6 +399,12 @@ s2n-tls provides multiple different methods to get the TLS protocol version of t
 
 `s2n_config` objects are used to change the default settings of a s2n-tls connection. Use `s2n_config_new()` to create a new config object. To associate a config with a connection call `s2n_connection_set_config()`. A config should not be altered once it is associated with a connection as this will produce undefined behavior. It is not necessary to create a config object per connection; one config object should be used for many connections. Call `s2n_config_free()` to free the object when no longer needed. _Only_ free the config object when all connections using it have been freed.
 
+Creating configs with `s2n_config_new()` can have performance implications at config-creation time
+due to loading default system certificates into the trust store (see 
+[Configuring the Trust Store](#configuring-the-trust-store)). If these system certificates are not
```

Comment:
```suggestion
[Configuring the Trust Store](#configuring-the-trust-store)).`s2n_config_new_minimal()` can be used instead to create configs without loading default system certificates.
```
Trying to make this sound better?

Reply:
Bleh this is kind of hard to phrase. But essentially I'm trying to switch your sentences to be more active rather than passive.